pages/api/verify.ts

```diff
@@ -17,7 +17,7 @@ const verifyReCaptchaToken = async (req: NextApiRequest, res: NextApiResponse) =
       res.status(404).json({ error: "Bad request" });
     }
   } catch (err) {
-    res.status(500).json({ error: "Malformed API Request" });
+    res.status(400).json({ error: "Malformed API Request" });
```

Comment:
In this case if an error is thrown in the axios call it is an internal server error.